openupgradelib/openupgrade_merge_records.py

```diff
@@ -365,6 +366,10 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if field.type in ('char', 'text', 'html'):
             if not op:
                 op = 'other' if field.type == 'char' else 'merge'
+            if op == 'first_not_null':
+                _list = filter(lambda x: x is not False, _list)
```

Comment:
Is it possible to have `""` instead of False?

Reply:
Yeah, `""` cases are excluded now.